src/@types/auth.d.ts

```diff
@@ -1,10 +1,8 @@
 export interface JwtPayload {
   id: string;
   email: string;
-}
-
-export interface JwtPayloadWithRT extends JwtPayload {
-  refreshToken: string;
+  iat: number;
+  exp: number;
```

Comment:
How about using readable names?

Reply:
In type JwtPayload (jsonwebtoken), use `iat`, `exp`.

```ts
// in jsonwebtoken index.d.ts
export interface JwtPayload {
    [key: string]: any;
    iss?: string | undefined;
    sub?: string | undefined;
    aud?: string | string[] | undefined;
    exp?: number | undefined;
    nbf?: number | undefined;
    iat?: number | undefined;
    jti?: string | undefined;
}
```

Is it better that custom JwtPayload extends JwtPayload in jsonwebtoken?

Reply:
Oh I think using `iat` and `exp` is better if so.